src/LookupResult/CprLookupResult.php

```diff
@@ -502,13 +619,13 @@ public function setBirthDate(\DateTime $birthDate): void {
   /**
    * Checks if the provided CPR is in list of guardians.
    *
-   * @param $cpr
+   * @param string $cpr
    *   Cpr number.
    *
    * @return bool
-   *   TRUE if person has provided CPR as guardian, FALSE otherwise.
+   *   TRUE if a person has provided CPR as guardian, FALSE otherwise.
```

Comment:
```suggestion
   *   TRUE if the person has provided CPR as guardian, FALSE otherwise.
```

or (the original):

```suggestion
   *   TRUE if person has provided CPR as guardian, FALSE otherwise.
```